docs/compilers/CSharp/Compiler Breaking Changes - DotNet 10.md

```diff
@@ -84,9 +84,14 @@ static class MemoryMarshal
 }
 ```
 
+### `Enumerable.Reverse`
+
```

Comment:
Can we add an example here of before / after on how to fix to this section? Effectively moving to `Enumerable.Reverse` explicitly? Want this to be a complete section for people to see.